src/chips/nrf51822/peripheral_interrupts.rs

```diff
@@ -0,0 +1,31 @@
+// Automatically generated by tools/nRF51_codegen.py
```

Comment:
Well, this file is created by the nrf51_codegen script. It basically goes over a XML file which describes the platform and creates a listing of the interrupts and registers. It doesn't do so for UART though, so I switched to using the nvic file and writing in the register names myself.
However I am not really using interrupts anyway